tools/code-owners-parser/ci.yml

```diff
@@ -26,10 +26,6 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml
   parameters:
     ToolDirectory: tools/code-owners-parser
-    # As of 12/11/2022 this is a temporary override until the globally-set
-    # SDK is set to net6.0. For details, please see:
-    # https://github.com/Azure/azure-sdk-tools/pull/4916
-    DotNetCoreVersion: 6.0.403
     # As of 12/11/2022 temporarily disabling TestPostSteps because the
```

Comment:
Can these test steps be re-enabled?

Reply:
I will enable them in a separate PR. I have a work item for it: https://github.com/Azure/azure-sdk-tools/issues/4938. I believe I need to make the script [refer to a new package](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/get-codeowners.ps1#L4), which I will will build and refer to once this PR is merged (even though I should be able to update it even before merging the PR; just being extra-safe here).